yaks/README.md

```diff
@@ -665,11 +678,42 @@ yaks.call(array_of_widgets, mapper: MyCollectionMapper, item_mapper: WidgetMappe
 ```
 
 If the mapper is left unspecified, Yaks will inspect whatever you pass
-it, and try various constant lookups based on naming. These all happen
+it. First it will test the given object against the mapping defined using `mapper_for`.
+If no mapper is found, it wil call `derive_mapper_from_item` or `derive_mapper_from_collection`
+depending on whether the given object is a collection or not. If the object responds
+to `to_ary` it is considered a collection.
+
+Notes:
+* you may override one of these two methods independently from the other.
+* this function takes either a symbol or any instance
```

Comment:
I wouldn't dwell on the `model_or_symbol`. It can be pretty much anything, as long as you have a derivation rule and a mapper that knows how to handle it